pom.xml

```diff
@@ -17,7 +17,7 @@
   <properties>
     <maven.compiler.source>21</maven.compiler.source>
     <maven.compiler.target>21</maven.compiler.target>
-    <spring.boot.version>3.4.2</spring.boot.version>
+    <spring.boot.version>3.4.5</spring.boot.version>
```

Comment:
spring boot 3.5.5 is out, that might fix the need to pin the other dependencies

Reply:
👍 will try moving to that and seeing if it simplifies things.

Reply:
Yep, updated and that allowed me to remove a number of those pinned dependencies. Thanks.